examples/stable/mssql/README.md

```diff
@@ -0,0 +1,352 @@
+# Microsoft SQL Server
+
+MS SQL Server is a relational database management system (RDBMS) developed by Microsoft. This product is built for the basic function of storing retrieving data as required by other applications. 
+It can be run either on the same computer or on another across a network.
+
+## Introduction
+This document will cover how to install SQL Server and how to run backup/restore actions.
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
```

Comment:
Please update to a later version. Kanister might not work with older versions. I assume we now need at least 1.17?
cc: @viveksinghggits